src/components/ActionButton/ActionButton.tsx

```diff
@@ -136,6 +141,7 @@ const ActionButton = ({
     >
       {showIcon ? (
         <Icon
+          aria-label={showLoader ? Label.Waiting : Label.Success}
```

Comment:
I guess it's good for the start, but ideally we will likely need to be able to pass a custom labels for loading and success here.

Reply:
But I don't mean it for this PR, but something to consider in future.